src/pages/app/posting_project_user/ProfilePage.js

```diff
@@ -43,6 +43,20 @@ const imageURL =
 const loremText =
   "Khi Beyond Birthday lấy mạng nạn nhân thứ ba của mình, hắn định làm một thử nghiệm—để xem có cách nào làm một con người chết vì xuất huyết trong mà không cần phải hủy hoại một bộ phận cơ thể nào hay không. Cụ thể, hắn cho nạn nhân chìm vào trạng thái vô thức bằng thuốc mê; ...";
 
+//The button to add a project
+function AddProjectButton(props) {
+  const userType = props.userType;
+  if (userType !== "STUDENT")
```

Comment:
Should not use plain string like this since this can cause unexpected mutation typing error
There's already a constants in folder **constants** containing the string value. Pls import from that file and use them

Reply:
yea, will do that also. Was too lazy to import when first coded this :))))